docs/packages/intro.md

```diff
@@ -2,18 +2,160 @@
 title: OpenWRT Packages
 ---
 
+This getting started guide is a comprehensive cookbook that provides self-contained references for installing, creating, and configuring packages on Omega2 devices for practical applications. The guide includes a concise overview of key terms, such as packages, package manager, firmware, and package source code, offering a clear understanding of the essential concepts involved in working with Omega2 devices.
```

Comment:
Agreed, we can rephrase it as follows:

`This chapter provides an understanding on creating software packages for Omega2 devices. Furthermore, it also covers a brief explanation of the key terms that are essential to understand when dealing with OpenWRT packages.`

Please let me know if this works

Reply:
Reads really well. Since the chapter deals with packages and this article introduces the key terms, how about we spell that out:

This chapter provides an understanding on creating software packages for Omega2 devices. 
This article covers a brief explanation of the key terms that are essential to understand when dealing with OpenWRT packages.